hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -3152,6 +3152,7 @@ public Config setDataConnectionConfigs(Map<String, DataConnectionConfig> dataCon
      */
     @Beta
     public Config addDataConnectionConfig(DataConnectionConfig dataConnectionConfig) {
+        dataConnectionConfig.validate();
```

Comment:
you miss the validation in `setDataConnectionConfigs`

Reply:
864528a727a29a37e9d99fe1353876bef2b806ce